### PR TITLE
Create spaapplication.md

### PR DESCRIPTION
**Pull request by hamiltonha**

### CODE REVIEW

File: api-reference/beta/resources/spaapplication.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Specifies settings for a single-page application.
+
+## Properties
+
+| Property | Type | Description |
+|:---------|:-----|:------------|
+| logoutUrl | String | Specifies the URL that will be used by the Microsoft authorization service to sign out a user using [front-channel](https://openid.net/specs/openid-connect-frontchannel-1_0.html), [back-channel](https://openid.net/specs/openid-connect-backchannel-1_0.html) or SAML sign out protocols. |
```
Comment (Lauragra) [inline api-reference/beta/resources/spaapplication.md:22]: I don't see this property in the metadata and it's not represented in the JSON - verify?

Comment (hamiltonha) [reply api-reference/beta/resources/spaapplication.md:22]: Yeah I actually think this should be removed

**Review by Lauragra — APPROVED** [review]

**Comment by Lauragra** [comment]
@hamiltonha , please update your branch in your fork to sync with the master branch. I tried to make the update to the changelog for you in your branch, but for some reason it's showing as deleting content from the changelog file. I think if you update your branch we can remove the deletion; we just want to add the new entry.

**Comment by hamiltonha** [comment]
@Lauragra  ok I updated it (I think, lmk if I did something wrong)

**Comment by Lauragra** [comment]
> @Lauragra ok I updated it (I think, lmk if I did something wrong)

It didn't work. It still looks like the changelog file is getting deleted. I hope I didn't mess something up, I only added one new entry. Can you make sure that your master branch in your fork is up to date with the main repo, and then make sure that your branch is updated with master?

**Comment by Lauragra** [comment]
> @Lauragra yep thats what I did so it should be all up to date?

Can you try again? Your authoring branch is still behind the master branch of the main repo: 
image. In the future, it might be easier to set yourself up as a contributor and work in a branch directly in the main repo, instead of a fork.

**Pull request merged**